content/en/docs/ibm/create-cluster.md

```diff
@@ -128,12 +189,150 @@ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
+   
+   b) To create a new VPC, proceed as follows:
+
+    ```shell
+    $ ibmcloud is vpc-create my-vpc
+    Creating vpc my-vpc in resource group kubeflow under account IBM as ...
+                                                      
+    ID                                             r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx   
+    Name                                           my-vpc   
+    ...  
+    ```
+
+   **Save the ID in a variable `VPC_ID` as follows, so that we can use it later.**
+   
+   `export VPC_ID=r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx`
+
+4. Create or use an existing subnet:
+
+    a) To use an existing subnet:
+    
+    ```shell
+    $ ibmcloud is subnets
+    Listing subnets for generation 2 compute in all resource groups and region ...
+    ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
+    0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
+    ```
+   If the above list contains the subnet corresponding to your VPC, that can be used to deploy your cluster - note it's
+   ID.
+   
+    b) To create a new subnet
+      * List address prefixes and note the CIDR block corresponding to a Zone,
+   in below example, for Zone: `us-south-3` CIDR block is : `10.240.128.0/18`
+         ```shell
+          $ ibmcloud is vpc-address-prefixes $VPC_ID
+          
+          Listing address prefixes of vpc r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx under account IBM as user new@user-email.com...
+          ID                                          Name                                CIDR block        Zone         Has subnets   Is default   Created   
+          r006-xxxxxxxx-4002-46d2-8a4f-f69e7ba3xxxx   rising-rectified-much-brew          10.240.0.0/18     us-south-1   false         true         2021-03-05T14:58:39+05:30   
+          r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+          r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+         ```
+   * Now create a subnet as follows.
+   
+       ```shell
+       $ ibmcloud is subnet-create my-subnet $VPC_ID us-south-3 --ipv4-cidr-block "10.240.128.0/18"
+       Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                              
+       ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+       Name                my-subnet
+       ```
```

Comment:
```suggestion
        ```shell
        ibmcloud is subnet-create my-subnet $VPC_ID us-south-3 --ipv4-cidr-block "10.240.128.0/18"
        ```

        Example output:
        
        ```
        Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
                              
        ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
        Name                my-subnet
        ```
```